src/wovnio/wovnphp/Utils.php

```diff
@@ -153,6 +153,20 @@ private static function checkIgnoreRegex($uri, $store = null)
         return false;
     }
 
+    public static function wovnDiagnosticsEnabled($store, $header)
+    {
+        if (strpos($header->url, 'wovn_diagnostics=1') == false) {
```

Comment:
Nitpick: if the diagnostic is enabled by query, then could we use `$header->query` instead of `$header->url`?

Reply:
Good point.

Reply:
Did some investigation, the `$header->query` is not exactly a representation of the real query (we processed it right after header creation, and only keep what we specify in the wovn.ini settings). So I guess `$header->url` is the best way to do it.

Reply:
I see. I guess you are talking about this: https://github.com/WOVNio/WOVN.php/blob/e122cf0b90247e3de7bff50947bb2d47e9f8ed9d/src/wovnio/wovnphp/Headers.php#L79.
Then I agree tat using `$header->url`.